datumaro/plugins/yolo_detection_format/format.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from enum import Enum, auto
+from typing import Dict
+import re
+
+
+class YoloDetectionPath:
+    DEFAULT_SUBSET_NAME = "train"
+    MUST_SUBSET_NAMES = ["train"]
+    ALLOWED_SUBSET_NAMES = ["train", "valid", "test", "val"]
+    RESERVED_CONFIG_KEYS = ["backup", "classes", "names"]
+    META_FILE = "data.yaml"
+
+    @staticmethod
+    def _parse_config(path: str) -> Dict[str, str]:
+        with open(path, "r", encoding="utf-8") as f:
+            config_lines = f.readlines()
+
+        config = {}
+
+        for line in config_lines:
```

Comment:
Please use pyyaml to parse YAML file. Don't reinvent the wheel: https://pyyaml.org/wiki/PyYAMLDocumentation

Use safe_load method please.

Reply:
Resolved. Used `yaml.safe_load`